OutOfSchool/OutOfSchool.WebApi.Tests/Redis/CacheServiceTests.cs

```diff
@@ -85,14 +86,71 @@ public async Task GetOrAddAsync_WhenDataNotExistsOrExpired_ShouldSaveNewData()
     }
 
     [Test]
-    public async Task RemoveAsync_ShouldCallCacheRemove()
+    public async Task RemoveAsync_ShouldCallCacheRemoveOnce()
     {
         // Arrange & Act
         await cacheService.RemoveAsync("Example");
 
         // Assert
         distributedCacheMock.Verify(
-            c => c.RemoveAsync("Example", It.IsAny<CancellationToken>()),
+            c => c.Remove("Example"),
+            Times.Once);
+    }
+
+    [Test]
+    public async Task ReadAsync_WhenDataExistsInCacheAndNotExpired_ShouldReturnData()
+    {
+        // Arrange
+        var expected = new Dictionary<string, string>()
+        {
+            {"ExpectedKey", "ExpectedValue"},
+        };
+        distributedCacheMock.Setup(c => c.Get(It.IsAny<string>()))
+            .Returns(Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(expected)));
+
+        // Act
+        var result = await readWriteCacheService.ReadAsync("ExpectedKey");
+
+        // Assert
+        result.Should().Contain("ExpectedValue");
+        distributedCacheMock.Verify(
```

Comment:
(Mock calls verification) In general, please prefer `VerifyAll` over specific `Verify(..., Times.Once)`. Former lets you check that absence of excessive `Setup`s

Reply:
Ok, done.
Used VerifyAll() method for tests instead of Verify().